core/iwasm/libraries/thread-mgr/thread_manager.c

```diff
@@ -240,6 +240,7 @@ void
 wasm_cluster_cancel_all_callbacks()
 {
     traverse_list(destroy_callback_list, free_node_visitor, NULL);
+    memset(destroy_callback_list, 0, sizeof(*destroy_callback_list));
```

Comment:
Had better use `bh_list_init` API as we can't assume the list implementation from this file

```suggestion
    bh_list_init(destroy_callback_list);
```

Reply:
OK, thank you.